ide/slf4j.api/external/slf4j-api-1.7.36-license.txt

```diff
@@ -1,10 +1,10 @@
 Name: SLF4J
 Version: 1.7.36
-License: MIT-slf4j
+License: MIT-slf4j-22
```

Comment:
The source license for 1.7.36 is here:
https://raw.githubusercontent.com/qos-ch/slf4j/v_1.7.36/LICENSE.txt

Reply:
The file `nbbuild/licenses/MIT-slf4j` could be removed. Not really a problem, but cleaning up now would IMHO make sense.

Reply:
Ok.